Mathlib/Data/Set/Quot.lean

```diff
@@ -29,49 +29,51 @@ instance : SetLike (Quot r) α where
       rw [← h]
       exact x.out_eq
 
+@[simp] lemma Quot.mem_toSet {x : α} {c : Quot r} : x ∈ c ↔ Quot.mk r x = c := Iff.rfl
```

Comment:
Note that because of the way that `SetLike` works, this is _not_ about `toSet`, this is about `SetLike.coe`! It probably would be best to inline `{x | Quot.mk r x = c}` into the `SetLike` instance so that there is not a stray definition around.
```suggestion
@[simp] lemma Quot.mem_coe {x : α} {c : Quot r} : x ∈ c ↔ Quot.mk r x = c := Iff.rfl
```